### DISCUSSION TITLE
No ACCESS option on internal module

### DISCUSSION THREAD

**Discussion by NealM722** [EdgeTX/edgetx#4255] [General]
Hello
I've just updated to EDGE TX on my FrSky Taranis QX7 Access  TX..  When I scroll down to the internal RF option, I am only given the option to select "XJT"..  There is no "ACCESS" option, which there used to be when I was using OpenTX..??

Can anyone advise where I've gone wrong ??? OR a possible solution ??

Here's what I've tried so far :
1. Flash firmware back to Open TX . .  when I do this I get the ability to select ACCESS on the internal RF module. !! ??
2. When I go back to Edge TX again, I again loose the "ACCESS" option on the internal RF
3. I've downloaded and installed the latest version of the internal module firmware from the FRSky website, without success.

Thanks in advance..

**Comment by pfeerick** [comment]
Go to SYS -> HARDWARE, and down the bottom, change the internal RF type
from XJT to ISRM. You should then be able to switch between OFF and ISRM
for the internal RF.

**Comment by NealM722** [comment]
Hello

Many thanks for your reply . . .  I'm pleased to report that this worked ..


AGAIN , THANK YOU...

**Reply by pfeerick** [reply]
No problem, glad it's working for you now! 🥳

**Comment by NealM722** [comment]
Would you believe I spent most of yesterday trying to resolve that !!!!

I'm very familiar with computer software, so was fairly sure it was going to be something that I'd missed, and also (probably); something fairly easy.   But I just couldn't see it . . .


Thanks again..
________________________________
